app/Listeners/LoginListener.php

```diff
@@ -4,6 +4,9 @@
 
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Queue\InteractsWithQueue;
+use App\Http\Controllers\LocationController;
+use Illuminate\Support\Facades\Route;
+use Illuminate\Http\Request;
 
```

Comment:
Route and Request imports might not be needed after making changes listed below.